pages/dao/[symbol]/members/Members.tsx

```diff
@@ -16,14 +16,15 @@ import PaginationComponent from '@components/Pagination'
 import useWalletOnePointOh from '@hooks/useWalletOnePointOh'
 import { useMembersQuery } from '@components/Members/useMembers'
 
-const Members = () => {
+const Members = ({ isNftMode }: { isNftMode: boolean }) => {
```

Comment:
lets not prop drill `isNftMode`. components can just figure it out for themselves and act accordingly.